zeroconf/src/macos/browser.rs

```diff
@@ -250,7 +256,23 @@ unsafe fn handle_get_address_info(
         .into());
     }
 
+    // on macOS the bytes are swapped for the port
+    #[cfg(target_os = "linux")]
+    let port :u16 = ctx.resolved_port;
```

Comment:
Bit of a formatting faux-pas, could you please run `cargo fmt` on the project.

Reply:
done